### DISCUSSION TITLE
can we reproduce/synthesize proofs of non properties

### DISCUSSION THREAD

**Discussion by chadnash** [pi-base/data#443] [Q&A] [answered]
sometimes non properties eg https://topology.pi-base.org/spaces/S000070/properties/P000010 simply refer to "counterexamples in topology"  and don't  have a explicit counter example.
Now i have two copies of counter examples and usually carry one in in my satchel but not on my current holiday. Now I know the example above is simple but it would be good to have these for completeness.

**Comment by prabau** [comment, marked as answer]
Sorry, but I don't think we are going to do that.  Our guiding principle is to give references to the literature, but not necessarily repeat the full argument here.  pi-base serves more as a guide to the literature than a full repository of it, which would not be realistic.

Furthermore, we have more important issues to deal with, including adding more properties, theorems, spaces and fixing existing problems, which you are welcome to contribute to. :-)

**Reply by chadnash** [reply]
thanks

**Comment by StevenClontz** [comment]
I agree with @prabau, with the suggestion that if you're seeking more detail on an assertion in pi-Base, using a forum like https://math.stackexchange.com/ to air out the details is an option, and we can add a link to such a discussion as a reference on pi-Base.
